src/containers/my-quizzes/selectable-question-quiz-view/SelectableQuestionQuizView.tsx

```diff
@@ -1,8 +1,12 @@
 import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
 import Box from '@mui/material/Box'
+import AppButton from '~/components/app-button/AppButton'
 import Typography from '@mui/material/Typography'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import ArrowForward from '@mui/icons-material/ArrowForward'
 
-import { Question } from '~/types'
+import { Question, ButtonVariantEnum, SizeEnum } from '~/types'
 import { styles } from '~/containers/my-quizzes/selectable-question-quiz-view/SelectableQuestion.styles'
 import SelectableQuestion from '~/containers/my-quizzes/selectable-question/SelectableQuestion'
```

Comment:
It will be good practice to organize our imports. Keep the `@mui` imports together, and move the `component` imports below them.